.rubocop.yml

```diff
@@ -1,4 +1,14 @@
-inherit_from: config/.base_rubocop.yml
+inherit_gem:
+  fablicop:
+    - 'config/.base_rubocop.yml'
```

Comment:
This `.rubocop.yml` is written for `fablicop` itself, and I think `inherit_from` is better than `inherit_gem` because we can try our new configuration without uploading our gem.